i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mssql.md

```diff
@@ -231,71 +267,508 @@ yum install centreon-plugin-Applications-Databases-Mssql
+| LOGSFILESMAXSIZEUNLIMITED       | Overload only unlimited autogrowth log files max size (in MB)                                       |                   |             |
+| WARNINGDATAFILESSPACEUSAGE      | Thresholds                                                                                          |                   |             |
+| CRITICALDATAFILESSPACEUSAGE     | Thresholds                                                                                          |                   |             |
+| WARNINGDATAFILESSPACEUSAGEFREE  | Thresholds                                                                                          |                   |             |
+| CRITICALDATAFILESSPACEUSAGEFREE | Thresholds                                                                                          |                   |             |
+| WARNINGDATAFILESSPACEUSAGEPRCT  | Thresholds                                                                                          |                   |             |
+| CRITICALDATAFILESSPACEUSAGEPRCT | Thresholds                                                                                          |                   |             |
+| WARNINGLOGFILESSPACEUSAGE       | Thresholds                                                                                          |                   |             |
+| CRITICALLOGFILESSPACEUSAGE      | Thresholds                                                                                          |                   |             |
+| WARNINGLOGFILESSPACEUSAGEFREE   | Thresholds                                                                                          |                   |             |
+| CRITICALLOGFILESSPACEUSAGEFREE  | Thresholds                                                                                          |                   |             |
+| WARNINGLOGFILESSPACEUSAGEPRCT   | Thresholds                                                                                          |                   |             |
+| CRITICALLOGFILESSPACEUSAGEPRCT  | Thresholds                                                                                          |                   |             |
+| EXTRAOPTIONS                    | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) | --verbose         |             |
+
+</TabItem>
+<TabItem value="Deadlocks" label="Deadlocks">
+
+| Macro        | Description                                                                                         | Valeur par défaut | Obligatoire |
+|:-------------|:----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTER       | Filter database to check                                                                            |                   |             |
```

Comment:
Because of this :
```    foreach my $row (@{$query_result}) {
        next if (defined($self->{option_results}->{filter_database}) && $self->{option_results}->{filter_database} ne ''
                    && $$row[0] !~ /$self->{option_results}->{filter_database}/);
        $self->{deadlocks}->{value} += $$row[1];
    }
```
I think it is a database name filter